azerothcore/azerothcore-wotlk
# fix(CORE/Crash): reload page GetSession check to avoid crash

## Changes Proposed:
-  Add check before sending notifications as console does not have a session
- 
## Issues Addressed:

- Closes Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7100

## Tests Performed:

-  Compile without error
- Started server and performed .reload page in console and as gm in game


## How to Test the Changes:

1. Compile
2. Start world server
3. Issue reload page from console
4. Log into gm character
5. Issue .reload page 
6. Check no crash and in game you got notification in middle of screen



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
